test/BUILD.bazel

```diff
@@ -30,3 +30,22 @@ cc_test(
         "@com_github_grpc_grpc//examples/protos:keyvaluestore",
     ],
 )
+
+cc_test(
+    name = "streaming",
+    timeout = "short",
+    srcs = [
+        "streaming.cc",
+        "test.h",
+    ],
+    # NOTE: need to add 'linkstatic = True' in order to get this to
+    # link until https://github.com/grpc/grpc/issues/13856 gets
```

Comment:
I've added a PR that might this obsolete by bumping grpc to a more recent version.

Reply:
Great! We can try it out and remove this in a subsequent PR if so. 🙂